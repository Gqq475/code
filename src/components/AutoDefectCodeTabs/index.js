import AutoDefectCodeTabs from './AutoDefectCodeTabs'
export default AutoDefectCodeTabs
