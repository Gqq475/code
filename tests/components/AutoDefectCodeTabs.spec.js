import React from 'react'
import AutoDefectCodeTabs from 'components/AutoDefectCodeTabs/AutoDefectCodeTabs'

describe('(Component) AutoDefectCodeTabs', () => {
  it('render normally', () => {
    const wrapper = shallow(
      <AutoDefectCodeTabs />
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
