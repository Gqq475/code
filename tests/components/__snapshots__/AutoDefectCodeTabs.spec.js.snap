// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) AutoDefectCodeTabs render normally 1`] = `
<div
  className="AutoDefectCodeTabs-container"
>
  <Tabs
    initialSelectedTabIndex={0}
  >
    <TabList
      className="tabs"
    >
      <Tab
        isDisabled={false}
        isSelected={false}
      >
        Unpack
      </Tab>
      <Tab
        isDisabled={false}
        isSelected={false}
      >
        LS
      </Tab>
      <Tab
        isDisabled={false}
        isSelected={false}
      >
        3L
      </Tab>
      <Tab
        isDisabled={false}
        isSelected={false}
      >
        NCD
      </Tab>
      <Tab
        isDisabled={false}
        isSelected={false}
      >
        NP
      </Tab>
      <Tab
        isDisabled={false}
        isSelected={false}
      >
        GE
      </Tab>
    </TabList>
    <TabPanel
      className="tabPanels"
    >
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNPPR0
        </span>
        <span>
          PH责PR残留
        </span>
      </div>
    </TabPanel>
    <TabPanel
      className="tabPanels"
    >
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNWPR0
        </span>
        <span>
          WET责PR残留
        </span>
      </div>
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNPLR0
        </span>
        <span>
          PH责LS残留
        </span>
      </div>
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNSLR0
        </span>
        <span>
          PVD责LS残留
        </span>
      </div>
    </TabPanel>
    <TabPanel
      className="tabPanels"
    >
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNPLR0
        </span>
        <span>
          PH责LS残留
        </span>
      </div>
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNWLR0
        </span>
        <span>
          WET责LS残留
        </span>
      </div>
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNCSR0
        </span>
        <span>
          Poly部责P-Si残留-key
        </span>
      </div>
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNPSR0
        </span>
        <span>
          PH责 P-Si残留-key
        </span>
      </div>
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNPSR1
        </span>
        <span>
          PH责 P-Si残留
        </span>
      </div>
    </TabPanel>
    <TabPanel
      className="tabPanels"
    >
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNSGR0
        </span>
        <span>
          PVD责 Gate残留-key
        </span>
      </div>
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNSGR1
        </span>
        <span>
          PVD责 Gate残留
        </span>
      </div>
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNPGR0
        </span>
        <span>
          PH责 Gate残留-key
        </span>
      </div>
    </TabPanel>
    <TabPanel
      className="tabPanels"
    >
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNPGR0
        </span>
        <span>
          PH责 Gate残留-key
        </span>
      </div>
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNPGR1
        </span>
        <span>
          PH责 Gate残留
        </span>
      </div>
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNDGR0
        </span>
        <span>
          DRY责 Gate残留-key
        </span>
      </div>
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNDGR1
        </span>
        <span>
          DRY责 Gate残留
        </span>
      </div>
    </TabPanel>
    <TabPanel
      className="tabPanels"
    >
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNSDR0
        </span>
        <span>
          PVD责 SD残留-key
        </span>
      </div>
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNWPR0
        </span>
        <span>
          WET责PR残留
        </span>
      </div>
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNSDR1
        </span>
        <span>
          PVD责 SD残留
        </span>
      </div>
      <div
        className="panel"
        onClick={[Function]}
      >
        <span>
          TNPDR0
        </span>
        <span>
          PH责 SD残留-key
        </span>
      </div>
    </TabPanel>
  </Tabs>
</div>
`;
